recipes/lora_finetune_distributed.py

```diff
@@ -139,12 +134,10 @@ def __init__(self, cfg: DictConfig) -> None:
         self.total_epochs = cfg.epochs
         self.max_steps_per_epoch = cfg.max_steps_per_epoch
         self.global_step = 0
-        self._resume_from_checkpoint = cfg.resume_from_checkpoint
+
```

Comment:
We should remove that check from the full distributed recipe then, since everything is >= 2.4 now (will not tackle that in this PR, that can be done separately)